src/DTPPodNode.cpp

```diff
@@ -67,14 +66,26 @@ namespace dunedaq {
     const OutputSinkNode& DTPPodNode::get_output_sink_node() const {
       return getNode<OutputSinkNode>("outsink");
     }
-    
 
-    void DTPPodNode::reset() const {
-    
+    std::map<std::string, uint32_t> DTPPodNode::get_firmware_config_info() const {
```

Comment:
Looking closer, this method uses only the info node.  So it should be a member of InfoNode.